config/application.rb

```diff
@@ -39,5 +39,6 @@ class Application < Rails::Application
 
     # Don't generate system test files.
     config.generators.system_tests = nil
+    config.autoload_paths += Dir[Rails.root.join('app', 'models', '**')]
```

Comment:
app/models should be in autoload paths by default. I think this is redundant?

Reply:
It seems like it didn't pick up the sub-directories unless it's added with **.

Reply:
PR to add namespaces to this branch: #900